rskj-core/src/test/java/co/rsk/peg/BridgeSupportProcessFundsMigrationTest.java

```diff
@@ -65,6 +67,16 @@ void processFundsMigration_in_migration_age_after_rskip_374_activation_testnet()
         test_processFundsMigration(BridgeTestNetConstants.getInstance(), activations, true);
     }
 
+    @Test
+    void processFundsMigration_in_migration_age_after_rskip_376_activation_testnet() throws IOException {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP146)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP357)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP374)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP376)).thenReturn(true);
```

Comment:
Could we use ActivationConfigForTest for this? I think that was the name, the class that returns an activations object per hard fork

Reply:
Just saw you created it for tbd600. Could we use it here?

Reply:
Can we use it now after the rebase?